Src/IronPython/Runtime/TypecodeOps.cs

```diff
@@ -188,7 +188,7 @@ class TypecodeOps {
                     return false; // All non-numeric types will not cause overflow.
             }
 
-            return value < minValue || value > maxValue;
+            return PythonOps.Compare(value, minValue) < 0 || PythonOps.Compare(value, maxValue) > 0;
```

Comment:
Unfortunately calling `Compare` is probably even worse (performance wise) than converting to a `BigInteger`. Also, since `Compare` is equivalent to calling `__cmp__` it will be removed from the codebase at some point.

Reply:
I tried benchmarking setting, and indeed Compare is much slower. The current change is only a little faster (~10%) than converting to BigInteger (possibly within the margin of error -- it wasn't a very empirical process), but I think it's a little cleaner than expecting a BigInteger in TypecodeOps. Here's what I did to benchmark:
```
import time
import array

def benchmark_memoryview(mv):
    start_time = time.time()
    for j in range(2000):
        for i in range(4000):
            mv[i] = i + j
    end_time = time.time()
    return end_time - start_time

mv = memoryview(array.array('i', [0]*4001)).cast('b')[1:16001].cast('i')
print('int unaligned', benchmark_memoryview(mv))

mv = memoryview(array.array('i', [0]*4000))
print('int aligned', benchmark_memoryview(mv))
```
Also, the aligned benchmark was super slow because `_shape` was null, so I initialized it in the constructor.